tools/ci_build/github/azure-pipelines/templates/react-native-ci.yml

```diff
@@ -198,6 +198,30 @@ stages:
       workingDirectory: '$(Build.SourcesDirectory)/js/react_native/e2e'
       displayName: Bootstrap Android and iOS e2e tests
 
+# TODO: remove this once we upgrade the RN to post 0.70.15 or 0.73.3+ versions this is just a temporary workaround
+    - script: |
+        if [[ "$OSTYPE" == "darwin"* ]]; then
+          sed -i .bak "/boostorg.jfrog.io/c\\
+        spec.source = { :http => 'https://archives.boost.io/release/1.76.0/source/boost_1_76_0.tar.bz2', }" boost.podspec
+        else
+          sed -i .bak "/boostorg.jfrog.io/c\spec.source = { :http => 'https://archives.boost.io/release/1.76.0/source/boost_1_76_0.tar.bz2', }" boost.podspec
+        fi
+        rm -f .bak
```

Comment:
You may remove this line if sed was not involved with ".bak"

Reply:
.bak is not needed for Linux